starling/src/starling/utils/RenderUtil.as

```diff
@@ -42,7 +42,10 @@ package starling.utils
                     Color.getRed(rgb)   / 255.0,
                     Color.getGreen(rgb) / 255.0,
                     Color.getBlue(rgb)  / 255.0,
-                    alpha);
+                    alpha,
+                    1.0, // depth
+					127  // stencil
```

Comment:
@Adolio Actually I've updated the code through the web page, the inconsistency arose there!!